### PR TITLE
http-prompt.md: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
A great http interactive tool.

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/http-prompt.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# http-prompt
+
+> An interactive command-line HTTP client featuring autocomplete and syntax highlighting.
+
+- Start with the last session or http://localhost:8000:
```
Comment (agnivade) [inline pages/linux/http-prompt.md]: The description should aim at users starting with a clean slate. Perhaps - `Launch a session targeting the default url of http://localhost:8000 or the previous session` sounds better.

File: pages/linux/http-prompt.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`http-prompt`
+
+- Start with the given URL:
```
Comment (agnivade) [inline pages/linux/http-prompt.md]: How about - `Launch a session with a given url`

File: pages/linux/http-prompt.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`http-prompt {{http://httpbin.org}}`
```
Comment (agnivade) [inline pages/linux/http-prompt.md]: Change httpbin.org to example.com to match with curl and wget pages.

File: pages/linux/http-prompt.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- Start with some initial options:
+
+`http-prompt {{localhost:8000/api}} --auth user:pass username={{somebody}}`
```
Comment (agnivade) [inline pages/linux/http-prompt.md]: Change "somebody" to "user"

File: pages/linux/http-prompt.md (hunk 1 above)
Comment (jsonbruce) [reply pages/linux/http-prompt.md]: Yeah, I used the examples from the official documentation.
Your description is much better. Thank you @agnivade

File: pages/linux/http-prompt.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+`http-prompt`
+
+- Launch a session with a given url:
+
+`http-prompt {{http://example.com}}`
+
+- Launch a session with some initial options:
+
+`http-prompt {{localhost:8000/api}} --auth user:pass username={{user}}`
```
Comment (sbrl) [inline pages/linux/http-prompt.md]: Not sure, but perhaps everything from `--auth` should be in a `{{}}` token?

Comment (agnivade) [reply pages/linux/http-prompt.md]: I think `user:pass` is part of the syntax. You enter the username and then you are prompted for the password.

Comment (jsonbruce) [reply pages/linux/http-prompt.md]: Yes, the `--auth` is the option key, and `user:pass` is the option value.
I've simplified the documentation.

**Comment by sbrl** [comment]
Ping @jsonbruce

**Comment by jsonbruce** [comment]
Copy that. I've fixed this error.

**Comment by sbrl** [comment]
Thanks, @jsonbruce! :D

**Pull request merged**